scripts/imagebuild.sh

```diff
@@ -23,9 +23,15 @@ dockerfile_dir="${base_dir}/images/federation-v2"
 
 [ -f "$base_dir/bin/controller-manager" ] || { echo "$base_dir/bin/controller-manager not found" ; exit 1 ;}
 
-if [[ "${TRAVIS_BRANCH}" != "master" ]]; then
-  echo "Not on master branch. Image build skipped." >&2
-  exit 0
+if [[ "${TRAVIS_TAG}" =~ ^[0-9]+\.[0-9]+\.[0-9]?\d$ ]]; then
+    echo "Pushing images with tag '${TRAVIS_TAG}'."
+    export TAG="${TRAVIS_TAG}"
```

Comment:
You don't need to export this; you can just say `tag=` and then reference elsewhere as `${tag}`